gravitee-apim-console-webui/src/portal/portal-settings-routing.module.ts

```diff
@@ -69,6 +72,44 @@ const portalRoutes: Routes = [
               },
             ],
           },
+          {
+            path: 'catalog',
+            component: PortalCatalogComponent,
+            children: [
+              {
+                path: '',
+                component: CategoryListComponent,
+                data: {
+                  permissions: {
+                    anyOf: ['environment-settings-r', 'environment-settings-u'],
+                  },
+                },
+              },
+              {
+                path: 'category/new',
+                component: CategoryCatalogComponent,
+                data: {
+                  permissions: {
+                    anyOf: ['environment-settings-r', 'environment-settings-u'],
+                  },
+                },
+              },
+              {
+                path: 'category/:categoryId',
+                component: CategoryCatalogComponent,
+                data: {
+                  permissions: {
+                    anyOf: ['environment-settings-r', 'environment-settings-u'],
+                  },
+                },
+              },
+            ],
+            data: {
+              permissions: {
+                anyOf: ['environment-settings-r', 'environment-settings-u'],
+              },
```

Comment:
🗑️  --> The children check the permissions for each page and the parent redirects to `''` by default.

Reply:
Done